src/main/java/org/generationcp/ibpworkbench/controller/HelpController.java

```diff
@@ -38,4 +40,10 @@ public String getUrl(@PathVariable final String helpDomain) {
 
 	}
 
+	@RequestMapping(value = "/exportFileName/{fileType}")
```

Comment:
Implement in a new class `GenerateFileNameController` with /**generateFileName**/{fileType}